core/ngw_tms_resources.py

```diff
@@ -48,10 +52,18 @@ def service_resource_id(self) -> int:
     def layer_params(self) -> Tuple[str, str, str]:
         connections_manager = NgwConnectionsManager()
         connection = connections_manager.connection(self.connection_id)
-        assert connection is not None
+        if connection is None:
+            raise NgwError(
+                "Can't get connection params", code=ErrorCode.PermissionsError
```

Comment:
There is a different code for that problem: `ErrorCode.NgwConnectionError` but I think this is also impossible situation